recipes/trgt/meta.yaml

```diff
@@ -1,35 +1,33 @@
 {% set name = "trgt" %}
-{% set version = "0.9.0" %}
-{% set trgt_sha256 = "fba2edb311c2ea4f15ae1e0f1b464ae470a73f803d12630b8ea6dbeeff3cc3f9" %}
-{% set trvz_sha256 = "08dcb1e5ca9d317140f26d635b323d12a8384b1263fda542e293effb2ca81280" %}
+{% set version = "1.0.0" %}
+{% set trgt_sha256 = "16f7892a3590a52ae2088dde9deb69da50e6748410ba446e92559f1d004e2c54" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
-source:
-  - url: https://github.com/PacificBiosciences/trgt/releases/download/v{{ version }}/trgt-v{{ version }}-linux_x86_64.gz
+source: 
+  - url: https://github.com/PacificBiosciences/trgt/releases/download/v{{ version }}/trgt-v{{ version }}-x86_64-unknown-linux-gnu.tar.gz
     sha256: {{ trgt_sha256 }}
-  - url: https://github.com/PacificBiosciences/trgt/releases/download/v{{ version }}/trvz-v{{ version }}-linux_x86_64.gz
-    sha256: {{ trvz_sha256 }}
 
 about:
   home: https://github.com/PacificBiosciences/trgt
-  license: BSD-3-Clause-Clear
+  license: Pacific Biosciences Software License (https://github.com/PacificBiosciences/trgt/blob/main/LICENSE.md)
   summary: Tandem repeat genotyping and visualization from PacBio HiFi data
 
-extra:
-  recipe-maintainers:
-    - zeeev
-    - williamrowell
-
 build:
   number: 0
   noarch: generic
```

Comment:
The downloaded tarball says linux-64 but here it says noarch. IMO this line should be removed